timeseries/src/main/scala/com/criteo/cuttle/timeseries/TimeSeriesScheduler.scala

```diff
@@ -140,7 +145,11 @@ case class TimeSeriesScheduler() extends Scheduler[TimeSeries] with TimeSeriesAp
           job -> (_state().apply(job) - Interval.closedOpen(start, end)))
         Right(id)
       }
+      (result, newBackfill)
     }
+    Database.createBackfill(newBackfill).transact(xa).unsafePerformIO
```

Comment:
You shouldn't log the backfill into the DB if it's invalid (partial periods, etc.)